ui/js/models/perf/alerts.js

```diff
@@ -34,6 +34,24 @@ treeherder.factory('PhAlerts', [
 
             return url;
         };
+        Alert.prototype.getSubtestsURL = function(alertSummary) {
+            var endpoint = '#/comparesubtest';
+            var urlParameters = {
+                framework: alertSummary.framework,
+                originalProject: alertSummary.repository,
+                originalSignature: this.series_signature.signature_hash,
+                newProject: alertSummary.repository,
+                newSignature: this.series_signature.signature_hash,
+            };
+            if (alertSummary.prevResultSetMetadata) {
+                urlParameters['originalRevision'] = alertSummary.prevResultSetMetadata.revision;
```

Comment:
Could you update this to `urlParameters.originalRevision`? Ditto with the statement just below.